Animal Wellfare and Euthanasia/SOP_BirdMonitoringAndTracking.md

```diff
@@ -0,0 +1,68 @@
+# SOP: Bird Monitoring and Tracking
+
+## Introduction
+
+This document aims to establish ground rules for monitoring and tracking the history and current state of birds. It covers procedures, whereabouts, health, reservations, the bird naming system, and interaction with animal facility workers. Any deviation from the standard procedures described in this document may result in harm to the birds, the experimenter's own research, or fellow lab members. Therefore, adhering to the ground rules is crucial.
+
+## Naming System
+
+Bird names are written in all capital letters. The name should begin with the letter "L," indicating the left leg where a ring should be placed. If there is no ring on the bird's left leg, use the right leg, and the first letter in the bird's name will be "R." If the bird has rings on both legs, its name will always start with the letter "L".
```

Comment:
It is not true that bird names use capital letters. In fact, various pieces of code will lower capital letters (to lowercase). 
It is important to say that in their personal files (add link) bird names use capital letters whereas elsewhere they do not or might not.